nexus-repository-manager/README.md

```diff
@@ -16,6 +16,10 @@
 
 As of October 24, 2023, we will no longer update or support this Helm chart.
 
+Deploying Nexus Repository in containers with an embedded database has been known to corrupt the database under some circumstances. We strongly recommend that you use an external PostgreSQL database for Kubernetes deployments. If you are deploying in AWS, you can use our [AWS Helm chart](https://github.com/sonatype/nxrm3-helm-repository/tree/main/nxrm-aws-resiliency) to deploy Nexus Repository in an EKS cluster.
+
+We do not currently provide Helm charts for on-premises deployments using PostgreSQL. Please see our [Installation and Upgrades Help Documentation](https://help.sonatype.com/repomanager3/installation-and-upgrades).
```

Comment:
Minor: I feel like a why is missing in the last sentence. e.g. for more info (probably that's not why).